### DISCUSSION TITLE
oasis引擎如何实现模型、模型动画资源的分别加载

### DISCUSSION THREAD

**Discussion by KingCyril** [galacean/discussion#27] [Engine] [answered]
![image](https://user-images.githubusercontent.com/85378736/199180838-05ab99ca-fb5c-426d-bf65-1b20a1b64267.png)
类似threejs这种AnimationMixer

**Comment by eyworldwide** [comment]
可以参考一下这个示例的代码：https://oasisengine.cn/0.8/examples#physx-controller

**Comment by luzhuang** [comment, marked as answer]
参考：https://galacean.antgroup.com/engine/docs/latest/cn/animation-animatorController#%E5%A4%8D%E7%94%A8%E5%8A%A8%E7%94%BB%E6%95%B0%E6%8D%AE  复用动画数据
